sdk/communication/azure-communication-rooms/src/test/java/com/azure/communication/rooms/RoomsTestBase.java

```diff
@@ -39,7 +39,7 @@ public class RoomsTestBase extends TestBase {
     protected static final TestMode TEST_MODE = initializeTestMode();
 
     protected static final String CONNECTION_STRING = Configuration.getGlobalConfiguration().get(
-                    "COMMUNICATION_LIVETEST_STATIC_CONNECTION_STRING",
+                    "COMMUNICATION_CONNECTION_STRING_ROOMS",
```

Comment:
How do we intend to test them on INT environment. Is there a way to run the pipeline before merging the PR?

Reply:
Yes, I have already tested each one of them on test pipelines. That is one of the reason for these pull request on Azure repo so that I can run internal test pipelines before merging

Reply:
Awesome. Is there a way to attach the pipeline link to the PR. Or add pipeline link to the description.

Reply:
It is already linked to this PR. You can visit them through the Checks tab on the top of the page.